### DISCUSSION TITLE
504 Error for Codespaces port-forwarding Ukraine region.

### DISCUSSION THREAD

**Discussion by AndriiTsok** [community/community#57336] [Codespaces] [answered]
### Select Topic Area

Bug

### Body

We started experiencing issues with Codespaces for all port forwarded URL display 504 errors opening URLs in Ukraine. The same URLs can be open in Canada and properly forward traffic.

![image](https://github.com/community/community/assets/733224/2084fbc0-5670-4837-ac97-90300c3737bb)

**Comment by ismaelcv** [comment]
We are experiencing the same in the EU region. 504 errors
Would be nice if thie Issue can  be looked out!

**Comment by jseiglie** [comment]
In Spain same problem happening

**Comment by dmgimatdinov** [comment]
Have the same problem.

**Comment by osortega** [comment, marked as answer]
Hello!
Sorry about the friction, we have fixed the issue.

You should be able to forward and connect to the port now

**Reply by ismaelcv** [reply]
Indeed! thank you!!

**Reply by AndriiTsok** [reply]
Thank you!

**Reply by dmgimatdinov** [reply]
Thank you!

**Comment by AndriiTsok** [comment]
@osortega hello, we started experiencing 502 and 503 again. The team is able to reach served web app from Canada but the same port forwarding URL at the same time does not work within Ukraine. Results in 502 or 503, no other evidence.
